tests/atomspace/TVMergeUTest.cxxtest

```diff
@@ -106,13 +106,16 @@ public:
             simpleTvs[i] = SimpleTruthValue::createTV(simpleMeans[i], simpleCounts[i]);
         }
 
+        MergeCtrl merge_ctrl(MergeCtrl::TVFormula::PLN_BOOK_REVISION,
+                             MergeCtrl::TVType::OLDER);
+
         // Merge between SimpleTruthValue objects
         //mean=(0.25*0.0 +0.5*1.0)/(0.0+1.0)=0.5 count=0.0+1.0-0.0*0.2=1.0
-        mergedTv = simpleTvs[0]->merge(simpleTvs[1]);
+        mergedTv = simpleTvs[0]->merge(simpleTvs[1], merge_ctrl);
```

Comment:
This isn't currently necessary since it uses the default anyway. If we change the default that test isn't gonna break.